tools/linter_lib/custom_check.py

```diff
@@ -190,10 +190,14 @@ def build_custom_checkers(by_lang):
     ]) + whitespace_rules
     python_rules = cast(RuleList, [
         {'pattern': '^(?!#)@login_required',
-         'description': '@login_required is unsupported; use @zulip_login_required'},
-        {'pattern': '".*"%\([a-z_].*\)?$',
-         'description': 'Missing space around "%"'},
-        {'pattern': "'.*'%\([a-z_].*\)?$",
+         'description': '@login_required is unsupported; use @zulip_login_required',
+         'good_lines': ['@zulip_login_required', '# foo @login_required'],
+         'bad_lines': ['@login_required', ' @login_required']},
+        {'pattern': '".*"%\(.*\)?$',
+         'description': 'Missing space around "%"',
+         'good_lines': ['\"%s\" % (\"foo\")', '\"%s\" % (foo)'],
+         'bad_lines': ['\"%s\"%(\"foo\")', '\"%s\"%(foo)']},
```

Comment:
Why all the `\`s` in the example lines here?  These should be e.g. `'"%s" % ("foo")', ''`

Reply:
I'm still not sure why you have all the `\`s here; you should be able to just use `"` and `'` strategically to avoid tons of escaping.

Reply:
Whoops, sorry, I thought I had that removed :/